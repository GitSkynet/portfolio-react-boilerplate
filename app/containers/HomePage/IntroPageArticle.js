import styled from 'styled-components';
import { rem } from 'polished';

const IntroPageArticle = styled.article`
  display: flex;
  flex-wrap: wrap;
  position: relative;
  width: 100%;

  @media (min-height: ${rem('424px')}) {
    align-items: center;
  }
`;

export default IntroPageArticle;
