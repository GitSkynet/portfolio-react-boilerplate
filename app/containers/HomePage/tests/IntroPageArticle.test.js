import React from 'react';
import { shallow } from 'enzyme';

import IntroPageArticle from '../IntroPageArticle';

describe('<IntroPageArticle />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = shallow(<IntroPageArticle />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
