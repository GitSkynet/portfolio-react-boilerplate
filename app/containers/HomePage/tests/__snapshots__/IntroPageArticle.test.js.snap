// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroPageArticle /> should render and match the snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <article
    className="IntroPageArticle__IntroPageArticle-guOsQo VvVDK"
/>,
  "nodes": Array [
    <article
      className="IntroPageArticle__IntroPageArticle-guOsQo VvVDK"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <IntroPageArticle__IntroPageArticle />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "VvVDK",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <article
          className="IntroPageArticle__IntroPageArticle-guOsQo VvVDK"
/>,
        "_debugID": 2,
        "_renderedOutput": <article
          className="IntroPageArticle__IntroPageArticle-guOsQo VvVDK"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <IntroPageArticle__IntroPageArticle />,
}
`;
